extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -784,6 +800,10 @@ public GracefulShutdownFilter createGracefulShutdownHandler() {
         return new GracefulShutdownFilter();
     }
 
+    private static boolean isLocalhost(String host) {
```

Comment:
Maybe to avoid confusion this method should be called `isWildCard` or something.